doc/src/manual/workflow-tips.md

```diff
@@ -64,8 +64,11 @@ line. A common pattern includes the following elements:
 
 ## Browser-based workflow
 
-It is also possible to interact with a Julia REPL in the browser via [IJulia](https://github.com/JuliaLang/IJulia.jl).
-See the package home for details.
+Here are a few ways to interact with the Julia REPL in a browser:
+
+  * One way is via [IJulia](https://github.com/JuliaLang/IJulia.jl).
+
+  * Another way is through [Pluto](https://github.com/fonsp/Pluto.jl) notebooks.
```

Comment:
```suggestion
There are a few ways to interact with Julia in a browser:
- [IJulia.jl](https://github.com/JuliaLang/IJulia.jl)
- [Pluto.jl](https://github.com/fonsp/Pluto.jl)
```

Reply:
Similar now to the list in `Profile`: https://raw.githubusercontent.com/JuliaLang/julia/master/doc/src/manual/profile.md

Reply:
@rikhuijzer I've made those changes. Is it fine now?

Reply:
@rikhuijzer any idea about when this MR will be merged?